1-exercises/A-accessing-values/exercise3.js

```diff
@@ -22,11 +22,15 @@ let basketballTeam = {
 
 // write code here
 
+let sortedNames = basketballTeam.topPlayers.sort();
+sortedNames.forEach((name) => {
+  console.log(name);
+});
 
```

Comment:
great job, you can also use forEach straight away instead of using a variable and then use forEach 
`basketballTeam.topPlayers.sort().forEach(name => console.log(name))`